hw11/Lesson11.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+import inspect
+
+func_name = None
+
+
+def my_decorator_func(func):
+    def deco_func():
```

Comment:
Use *args and **kwargs here to make in works with functions that take arguments. And pass these arguments to the func() call